### PR TITLE
feat: save PSM adminFacet in bootstrap

### LINKED ISSUE #6100: Save PSM adminFacet to enable big-hammer upgrade
## What is the Problem Being Solved?

In order for a javascript program running in the bootstrap environment to be able to upgrade the PSM (see #6099), the adminFacet will need to be available there.

## Description of the Design

Export the AdminFacet from the contractGovernor, and save it in the bootstrap environment.

## Security Considerations

POLA pushes us to want fine-grained access to the caps in bootstrap.

## Test Plan

See #6099

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
closes: #6100

## Description

Save the PSM adminFacet from bootstrap. Also, the contractGovernor will now make the adminFacet available when other contracts need it.

### Security Considerations

The admin facet is very powerful. The bootstrap environment is considered a secure context in which to hold powerful facets for the Econ Committee's use.

### Documentation Considerations

None.

### Testing Considerations

deferred to #6099.

### CODE REVIEW

File: packages/inter-protocol/src/proposals/econ-behaviors.js [hunk 1]
```diff
@@ -40,6 +40,15 @@ const sanitizePathSegment = name => {
   return candidate;
 };
 
+/**
+ * There's an obsolete version in utils.d.ts
```
Comment (turadg) [inline packages/inter-protocol/src/proposals/econ-behaviors.js]: please put these updates there instead so there's one correct one and no incorrect ones.
```ts
type AdminFacet = {
  getVatShutdownPromise: () => Promise<any>; // Completion, which is currently any
  upgradeContract: (contractBundleId: string, newPrivateArgs: any) => void;
  restartContract: (newPrivateArgs: any) => void;
};
```

File: packages/inter-protocol/src/proposals/startPSM.js [hunk 2]
```diff
@@ -138,14 +138,11 @@ export const startPSM = async (
     }),
   );
 
-  const governedInstance = await E(governorFacets.creatorFacet).getInstance();
-  const creatorFacet = E(governorFacets.creatorFacet).getCreatorFacet();
-
-  psmInstanceR.resolve(governedInstance);
+  psmInstanceR.resolve(await E(governorFacets.creatorFacet).getInstance());
```
Comment (turadg) [inline packages/inter-protocol/src/proposals/startPSM.js:141]: with all these facets, consider having a single entity in the space that is the result of `zoe.startInstance`.

**Review by turadg — CHANGES_REQUESTED** [review]

File: packages/inter-protocol/src/proposals/startPSM.js (hunk 2 above)
Comment (dckc) [reply packages/inter-protocol/src/proposals/startPSM.js:141]: yeah; we tried that... ran into problems around promises or something... the answer isn't obvious, so we agreed to defer.

File: packages/governance/src/contractGovernor.js [hunk 3]
```diff
@@ -281,6 +282,7 @@ const start = async (zcf, privateArgs) => {
     voteOnApiInvocation,
     voteOnOfferFilter: voteOnFilter,
     getCreatorFacet: () => limitedCreatorFacet,
+    getAdminFacet: () => adminFacet,
```
Comment (dckc) [inline packages/governance/src/contractGovernor.js:285]: This gave me some pause... we talked it over with @erights and agreed it's appropriate.

**Review by dckc — APPROVED** [review]
This looks like it does what we need for now.

File: packages/inter-protocol/src/proposals/econ-behaviors.js (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/proposals/econ-behaviors.js]: done

**Review by turadg — APPROVED** [review]

**Pull request merged**